File: src/main/java/com/web/model/dto/NotificationDTO.java
package com.web.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Timestamp;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor

public class NotificationDTO {
    private Long id;
    private String title;
    private String content;
    private Timestamp sentAt;
    private boolean isRead;
    private Long userEntityId;
}

File: src/main/resources/templates/admin/assets/js/model/dto/NotificationDTO.js
export class NotificationDTO {
    constructor(id, title, content, type, sendAt, isRead, userEntityId) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.type = type;
        this.sendAt = sendAt;
        this.isRead = isRead !== undefined ? isRead : false;
        this.userEntityId = userEntityId;
    }
}